src/main/java/Duke.java

```diff
@@ -1,10 +1,245 @@
+import java.util.*;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.time.*;
+import java.time.format.DateTimeFormatter;
+import java.text.SimpleDateFormat;
 public class Duke {
```

Comment:
Perhaps an additional empty line can help make your code cleaner?

```suggestion
import java.text.SimpleDateFormat;

public class Duke {
```